orchestrator/app.py

```diff
@@ -180,6 +187,26 @@ def register_subscription_models(product_to_subscription_model_mapping: Dict[str
         """
         SUBSCRIPTION_MODEL_REGISTRY.update(product_to_subscription_model_mapping)
 
+    def register_graphql(self: "OrchestratorCore", query: Any = Query, mutation: Any = Mutation) -> None:
+        strawberry_models = GRAPHQL_MODELS
+        strawberry_enums: EnumList = {}
+        products = {
+            product_type.__base_type__.__name__: product_type.__base_type__
+            for product_type in SUBSCRIPTION_MODEL_REGISTRY.values()
+            if product_type.__base_type__
+        }
+        for key, product_type in products.items():
+            add_class_to_strawberry(
+                model_name=graphql_subscription_name(key),
```

Comment:
What happens if `add_class_to_strawberry` is called for different product types but with the same `model_name=graphql_subscription_name(key)`?

Reply:
it will create the strawberry class twice and break it, adding a check to see if it exists in the `strawberry_models` dict